c2corg_ui/static/js/suggestiondirective.js

```diff
@@ -0,0 +1,50 @@
+goog.provide('app.suggestionDirective');
+
+/**
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {angular.$sce} $sce Angular Strict Contextual Escaping service.
+ * @return {angular.Directive} Directive Definition Object.
+ * @ngInject
+ */
+app.suggestionDirective = function($compile, $sce) {
+  var header = '<span class="suggestion-text">' +
```

Comment:
Can't we use a partial for this?
This would be clearer to isolate the HTML for the JS code.